glean-core/preview/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "glean-preview"
+version = "0.0.1"
+authors = ["Jan-Erik Rediger <jrediger@mozilla.com>", "The Glean Team <glean-team@mozilla.com>"]
+description = "Nice Glean Rust API"
+repository = "https://github.com/mozilla/glean"
+readme = "README.md"
+license = "MPL-2.0"
+edition = "2018"
+keywords = ["telemetry"]
```

Comment:
```suggestion
keywords = ["telemetry", "glean"]
```

?

Reply:
Glean isn't a keyword by itself yet (https://crates.io/keywords/glean), but I guess I should start adding it to the other crates.

Reply:
Ah! Will it be shown once more published packages start using the keyword?

Reply:
It should, yes

Reply:
Fine by me, then :)